src/mantarray_file_manager/exceptions.py

```diff
@@ -56,3 +57,26 @@ def __init__(self, file_version: str):
         super().__init__(
             f"There is no supported migration path from version {file_version}. Supported paths are: {FILE_MIGRATION_PATHS}."
         )
+
+
+class UnsupportedArgumentError(Exception):
+    """Error raised if the arguments indicating the amount of centimilliseconds to be trimmed from the start and end are both None."""
+
+    def __init__(self) -> None:
+        super().__init__("Both arguments cannot be None.")
+
+
+class TooTrimmedError(Exception):
+    """Error raised if the amount of centimilliseconds to be trimmed from the start and end exceeds the length of the recording."""
+
+    def __init__(self, from_start: int, from_end: int) -> None:
+        super().__init__(
+            f"When trimming {from_start} centimilliseconds from the start and {from_end} centimilliseconds from the end, the length of the recording is exceeded."
+        )
+
+
+class UnsupportedMantarrayFileVersionForTrimmingError(Exception):
```

Comment:
Perhaps `MantarrayFileNotLatestVersionError` would be more concise.  That way the error may also be applicable potentially in other contexts as well----as currently the only reason we're not "supporting" a file version for trimming is just because we want to make sure people migrate first